mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/109 objectives tab look and feel

Updated UI on Objective View

This looks a little different than the concept art. The important thing is that it is understood by the player.

To test:

- Launch the introductory scenario
- complete the first phase (buy computers for Moe and Curly, Connect them to the network, then hire Larry the IT Guy)
- Press play in the Office view
- Open the Objectives tab to see the objectives and how they look when they are incomplete and complete

Closes #109
